README.md

```diff
@@ -45,8 +44,7 @@ tomorrowAsString(new Date('2015-01-01')); // '2015-01-02';
 
 ## API
 
-### parse
-`String -> Date`
+### parse :: String -> Date
```

Comment:
It's worth noting that you'll lose nice anchors. This'll become [#parse--string---date][1], for example. [Transcribe][2] provides a partial solution to this problem.

On a different note, I suggest wrapping the heading in backticks:

```diff
-### parse :: String -> Date
+### `parse :: String -> Date`
```


[1]: #parse--string---date
[2]: https://github.com/plaid/transcribe